example/driver/simpledriver.go

```diff
@@ -154,6 +157,13 @@ func (s *SimpleDriver) HandleWriteCommands(deviceName string, protocols map[stri
 				err := fmt.Errorf("SimpleDriver.HandleWriteCommands; the data type of parameter should be Int32, parameter: %s", params[i].String())
 				return err
 			}
+		case "Uint8Array":
+			v, err := params[i].Uint8ArrayValue()
+			if err == nil {
+				fmt.Println("Uint8 array value from http request: ", v)
```

Comment:
why don't you use lc instead of fmt?

Reply:
please state write command instead of http request, because the command request might not come from http in the future